readme.md

```diff
@@ -180,6 +183,15 @@ Please talk to the OpenRCT2 team first before starting to develop a new feature.
 ## 4.3 Translation
 You can translate the game into other languages by editing the language files in ```data/language``` directory. Please join discussions and submit pull requests to [OpenRCT2/Localisation](https://github.com/OpenRCT2/Localisation).
 
+## 4.4 Graphics
+You can help create new graphics for the game by visiting the [OpenGFX project](https://gitter.im/zrowny/OpenRCT2-OpenGraphics). 3D Modellers needed!
```

Comment:
I would write OpenGraphics instead of OpenGFX, since the latter is otherwise too easily confused with its OpenTTD counterpart.